src/api/routes.ts

```diff
@@ -58,28 +58,134 @@ export function logout(req: auth.AuthRequest, res: express.Response): express.Re
   }
 }
 
+export function getUser(req: auth.AuthRequest, res: express.Response): void | express.Response {
+  let auth = getAuth();
+  let user = auth.users
+    .find(u => u.name === req.params[0]);
+  if (user) {
+    return res.status(200).json({
+      _id: 'org.couchdb.user:admin', email: user.email, name: user.name });
+  } else {
+    return res.status(404).json('');
+  }
+}
+
 export function getPackage(req: auth.AuthRequest, res: express.Response): void | express.Response {
-  let baseUrl = req.protocol + '://' + req.get('host');
-  let pkgName: string = req.params.package;
-  let config = getConfig();
+  let request = req.headers['referer'].split(' ');
+  if (request[0] === 'owner') {
+    let pkgName: string = req.params.package;
+    let auth = getAuth();
+    let pkg = auth.packages.find(p => p.name === pkgName);
+    if (pkg) {
+      let usernames = pkg.owners;
+      let users = [];
```

Comment:
because in previous `.map` call if user doesn't exists you return `false`. 
`.filter(Boolean)` then "cleanup" that false values in the array and you get array with users only.